BrowserKit/Sources/Common/Theming/ThemeColourPalette.swift

```diff
@@ -97,4 +97,7 @@ public protocol ThemeColourPalette {
 
     // MARK: - Shadow
     var shadowDefault: UIColor { get }
+
+    // MARK: - Separators
+    var separatorDefault: UIColor { get }
```

Comment:
I don't see `separatorDefault` in the Mobile Style Figma, and therefore, this does not belong here. If we're looking to add it here, we should chat with Design or whomever is in charge of design system (possibly @cwzilla) about adding it.

Reply:
Yes `borderPrimary` is using LightGrey30 but only for LightMode.

Reply:
yes. Should it be the same colour everywhere? That's not advisable, because that's not how we're using colours in our app. So that would be a thing you need to chat with design on this. If you need help, please bring me in.